### ISSUE TITLE
Adding extension (µm units) to the axis

### ISSUE THREAD

**Issue by BenediktBurger** [rgeneaux/pymodaq_plugins_test_pylablib#6]
I think it would be great to add the extension of the sensor to the axis (so you get a size in µm/mm instead of Pixels).

See also https://github.com/BenediktBurger/pymodaq_plugins_basler/issues/7

I might propose it.

**Comment by rgeneaux** [comment]
Good idea, is the pixel size an attribute of the camera that can be read from the library in pylon? I will check if pylablib has that function.

**Comment by BenediktBurger** [comment]
As far as I know, you have to know the pixel size of your camera (but I asked on pypylon whether there is a possibility).

**Comment by BenediktBurger** [comment]
I think a user defined parameter (as an alternative) would be good.
